menu/menu.sh

```diff
@@ -6,7 +6,7 @@
 true
 SCRIPT_NAME="Main Menu"
 # shellcheck source=lib.sh
-source /var/scripts/fetch_lib.sh &>/dev/null || source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh) &>/dev/null
```

Comment:
Remove in both places please, the curl as well.

Reply:
As I wrote this is only an issue with fetch_lib.sh and not with lib.sh

Reply:
It was already fixed for fetch_libs.sh

Reply:
It's harder to debug without an output. Nothing is shown anyway if not run with `-x`.

Reply:
@enoch85 Yeas, this is why I added the /dev/null in the first place, because I think the information that is printed out by the lib, if running in -x mode is not worth the flooting of the console (I find it much harder to debug if the console gets floated)

Reply:
@enoch85 What is your opinion on my statement above?

Reply:
/devnull is now removed, so this is solved.